src/cascade/dismod/db/wrapper.py

```diff
@@ -17,14 +17,15 @@
 from sqlalchemy.exc import OperationalError, StatementError
 from sqlalchemy import Integer, String, Float, Enum
 
-from cascade.dismod.db.metadata import Base, add_columns_to_table, DensityEnum
+from cascade.dismod.db.metadata import Base, add_columns_to_table
+from cascade.dismod.constants import DensityEnum
 from cascade.dismod.db import DismodFileError
 
-from cascade.core.log import getLoggers
+from cascade.core import getLoggers
 CODELOG, MATHLOG = getLoggers(__name__)
 
 
-def _get_engine(file_path):
+def get_engine(file_path):
```

Comment:
Yes, this is as it should be. Probably makes sense to have a factory for the common `DismodFile(get_engine(Path("...")))` case too.